front/src/applications/stdcm/utils.ts

```diff
@@ -0,0 +1,34 @@
+// TODO: The number must be calculated from a hash of stdcm inputs (to have a stable number). It is currently generated randomly, so there could be duplicates.
+export function generateCodeNumber(): string {
+  const currentDate = new Date();
+  const year = currentDate.getFullYear().toString().substr(-2);
+  const month = (currentDate.getMonth() + 1).toString().padStart(2, '0');
+  const generateRandomString = (length: number) =>
+    [...Array(length)].map(() => Math.floor(Math.random() * 10)).join('');
```

Comment:
Move the function generateRandomString outside and above generateCodeNumber

Reply:
done